test/attributes_test.rb

```diff
@@ -71,7 +71,7 @@ def test_spatial_attributes
 
     assert_equal 3857, data.polygon.srid
     assert_equal 3857, data.path.srid
-    assert_equal data.path, data.polygon.exterior_ring
+    assert_equal data.path.to_s, data.polygon.exterior_ring.to_s
```

Comment:
was there an issue with this?

Reply:
yes,  the new release of rgeo is not using the same object. but it values are the same.

Reply:
hmm that seems weird. Do you know exactly what was causing the issue?

Reply:
latest version of rgeo causing the issue. I will check that upstream then i will update this test later.